README.md

```diff
@@ -65,10 +65,15 @@ Changing the ADF code results in the recompilation and hot-reloading of the ACA
 
 ## Unit Tests
 
-Use standard Angular CLI commands to test the projects:
+Use following command to test the projects:
 
 ```sh
-ng test <project>
+nx affected:test <project>
```

Comment:
as this section describes running test for a particular project, it should be: `nx text <project>`

Reply:
ah right, corrected